src/main/java/bookface/logic/parser/edit/EditBookArgumentsParser.java

```diff
@@ -0,0 +1,57 @@
+package bookface.logic.parser.edit;
+
+import static bookface.logic.parser.CliSyntax.PREFIX_AUTHOR;
+import static bookface.logic.parser.CliSyntax.PREFIX_TITLE;
+
+import bookface.commons.core.Messages;
+import bookface.commons.core.index.Index;
+import bookface.logic.commands.edit.EditBookCommand;
+import bookface.logic.parser.ArgumentMultimap;
+import bookface.logic.parser.ArgumentTokenizer;
+import bookface.logic.parser.Parseable;
+import bookface.logic.parser.ParserUtil;
+import bookface.logic.parser.exceptions.ParseException;
+
+/**
+ * Parses input arguments and creates the relevant new AddCommand object for the relevant entity to be added
+ */
+public class EditBookArgumentsParser implements Parseable<EditBookCommand> {
+    /**
+     * Parses the given arguments in the context of the DeleteCommand
```

Comment:
Should be `EditCommand` instead of `DeleteCommand`

Reply:
Fixed